src/main/java/uk/gov/companieshouse/api/testdata/model/entity/CompanyProfile.java

```diff
@@ -5,26 +5,165 @@
+        @Field("next_due")
+        private Instant accountsNextDue;
+        @Field("next_accounts.period_start_on")
+        private Instant periodStart;
+        @Field("next_accounts.period_end_on")
+        private Instant periodEnd;
+        @Field("next_accounts.due_on")
+        private Instant nextAccountsDueOn;
+        @Field("next_accounts.overdue")
+        private Boolean nextAccountsOverdue;
+        @Field("next_made_up_to")
+        private Instant accountsNextMadeUpTo;
+        @Field("accounting_reference_date.day")
+        private String accountingReferenceDateDay;
+        @Field("accounting_reference_date.month")
+        private String accountingReferenceDateMonth;
+
+        public Instant getAccountsNextDue() {
+            return accountsNextDue;
+        }
+
+        public void setAccountsNextDue(Instant accountsNextDue) {
+            this.accountsNextDue = accountsNextDue;
+        }
+
+        public Instant getPeriodStart() {
+            return periodStart;
+        }
+
+        public void setPeriodStart(Instant periodStart) {
+            this.periodStart = periodStart;
+        }
+
+        public Instant getPeriodEnd() {
+            return periodEnd;
+        }
+
+        public void setPeriodEnd(Instant periodEnd) {
+            this.periodEnd = periodEnd;
+        }
+
+        public Instant getNextAccountsDueOn() {
+            return nextAccountsDueOn;
+        }
+
+        public void setNextAccountsDueOn(Instant nextAccountsDueOn) {
+            this.nextAccountsDueOn = nextAccountsDueOn;
+        }
+
+        public Boolean getNextAccountsOverdue() {
+            return nextAccountsOverdue;
+        }
+
+        public void setNextAccountsOverdue(Boolean nextAccountsOverdue) {
+            this.nextAccountsOverdue = nextAccountsOverdue;
+        }
+
+        public Instant getAccountsNextMadeUpTo() {
+            return accountsNextMadeUpTo;
+        }
+
+        public void setAccountsNextMadeUpTo(Instant accountsNextMadeUpTo) {
+            this.accountsNextMadeUpTo = accountsNextMadeUpTo;
+        }
+
+        public String getAccountingReferenceDateDay() {
+            return accountingReferenceDateDay;
+        }
+
+        public void setAccountingReferenceDateDay(String accountingReferenceDateDay) {
+            this.accountingReferenceDateDay = accountingReferenceDateDay;
+        }
+
+        public String getAccountingReferenceDateMonth() {
+            return accountingReferenceDateMonth;
+        }
+
+        public void setAccountingReferenceDateMonth(String accountingReferenceDateMonth) {
+            this.accountingReferenceDateMonth = accountingReferenceDateMonth;
+        }
+    }
+
+    public class ConfirmationStatement {
+        @Field("next_made_up_to")
+        private Instant confirmationStatementNextMadeUpTo;
+        @Field("overdue")
+        private Boolean confirmationStatementOverdue;
+        @Field("next_due")
+        private Instant confirmationStatementNextDue;
```

Comment:
Being in its own object, these variables won't need to be prefixed `confirmationStatement`